osmnx/downloader.py

```diff
@@ -670,14 +679,14 @@ def overpass_request(data, pause=None, error_pause=60):
     else:
         # if this URL is not already in the cache, pause, then request it
         if pause is None:
-            this_pause = _get_pause(base_endpoint)
-        utils.log(f"Pausing {this_pause} seconds before making HTTP POST request")
-        time.sleep(this_pause)
+            this_pause = _get_pause(session, base_endpoint)
+            utils.log(f"Pausing {this_pause} seconds before making HTTP POST request")
+            time.sleep(this_pause)
```

Comment:
These need to be moved back out of the `if` block.

Reply:
That produces an out of scope variable reference to get_pause similar to the recently fixed issue for the response status code.